mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -35,6 +40,7 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
   const [refreshing, setRefreshing] = useState(false)
   const dispatch = useDispatch()
   const { wallets } = useWalletPersistenceContext()
+  const { isLight } = useThemeContext()
```

Comment:
Just call this on the component instead of passing it

Reply:
updated